lib/runtime-core/src/instance.rs

```diff
@@ -63,7 +63,16 @@ impl Instance {
         // Initialize the vm::Ctx in-place after the backing
         // has been boxed.
         unsafe {
-            *inner.vmctx = vm::Ctx::new(&mut inner.backing, &mut inner.import_backing, &module)
+            *inner.vmctx = match imports.call_state_creator() {
+                Some((data, dtor)) => vm::Ctx::new_with_data(
```

Comment:
`dtor`? maybe `destructor` or `finalizer`?

Reply:
Yeah, either of those is fine. `dtor` was just short and it's common in the c++ world.